_posts/2022-04-11-nwaku-update.md

```diff
@@ -0,0 +1,277 @@
+---
+layout: post
+name:  "What's new in `nwaku`?"
+title:  "What's new in `nwaku`?"
+date:   2022-04-11 10:00:00 +0200
+author: hanno
+published: true
+permalink: /nwaku-update-2022-04
+categories: research
+summary: A summary of recent developments in the Nim-based Waku v2 client and a preview of current and future focus areas.
+image: /assets/img/vac.png
+discuss:
+---
+
+## Background
+
+If you've been following our [research log](https://vac.dev/research-log/),
+you'll know that many things have happened in the world of Waku v2 since [our last general update](./2020-02-14-waku-update.md).
+In line with our [long term goals](https://vac.dev/#about),
+we've introduced new protocols,
+tweaked our existing protocols
+and expanded our team capacity.
+We've also shown [in a series of practical experiments](./2021-10-25-waku-v1-vs-waku-v2.md) that Waku v2 does indeed deliver on some of the [theoretical advantages](./2020-07-01-waku-v2-pitch.md) it was designed to have over its predecessor, Waku v1.
+A [sustainability and business workshop](https://forum.vac.dev/t/vac-sustainability-and-business-workshop/116) led to the formulation of a clearer vision for Vac as a team.
+
+From the beginning, our protocol development has been complemented by various client implementations of these protocols,
+first in [Nim](https://github.com/status-im/nim-waku),
+but later also in [JavaScript](https://github.com/status-im/js-waku)
+and [go](https://github.com/status-im/go-waku).
+A follow-up post will clarify the purposes, similarities and differences between these three clients.
+The [Nim client](https://github.com/status-im/nim-waku/tree/d2fccb5220144893f994a67f2cc26661247f101f/waku/v2), is our reference implementation,
+developed by the research team in parallel with the specs
+and building on a home-grown implementation of [`libp2p`](https://github.com/status-im/nim-libp2p).
+We believe the Nim client is suitable to run as [a standalone adaptive node](./2020-07-01-waku-v2-pitch.md),
```

Comment:
Stronger phrase, not "believe" but "is"

Reply:
Done in [05b607c](https://github.com/vacp2p/vac.dev/pull/55/commits/05b607c2812a885741e7f51bc0b465aff978a26e)